web_external/HeaderBar/style.styl

```diff
@@ -1,73 +1,24 @@
 .v-header-wrapper
-  display flex
```

Comment:
Please do not replace the flex layout with floating elements.

Reply:
Because?